cli/src/commands/lock_files.rs

```diff
@@ -70,7 +70,7 @@ pub fn get_packages_from_lockfile(path: &str) -> Result<(Vec<PackageDescriptor>,
 }
 
 /// Get all packages for a specific lockfile type.
-fn parse<P: Parseable>(path: &Path) -> Result<(Vec<PackageDescriptor>, PackageType)> {
+pub(super) fn parse<P: Parseable>(path: &Path) -> Result<(Vec<PackageDescriptor>, PackageType)> {
```

Comment:
I'll change this to `pub`

I was hesitant to use `pub` here mostly because this `phylum-cli` crate is a lib crate and (in theory) may someday be published to `crates.io`. So a piece of me worries every time we add a new `pub` function that we're committing to a public API. But it's premature to worry about that. And we can switch these functions to `pub(crate)` if necessary in the future.